cloud_governance/common/clouds/aws/utils/utils.py

```diff
@@ -107,3 +107,25 @@ def iter_client_function(func_name: Callable, output_tag: str, iter_tag_name: st
                 resources = func_name(Marker=resources[iter_tag_name], **kwargs)
             resource_list.extend(resources[output_tag])
         return resource_list
+
+    @staticmethod
+    def is_cluster_resource(cluster_prefix: str, tags: list = None, tag: dict = None):
+        """
+        This method checks if the resource i.e: EC2, IAM is a cluster resource
+        Resources have the tags
+        :param tag:
+        :param cluster_prefix:
+        :param tags:
+        :return:
+        """
+        prefixes = cluster_prefix.strip().split(",")
+        for prefix in prefixes:
+            if tag:
+                if tag.get('Key').startswith(prefix):
+                    return True, 'kubernetes.io/cluster/' + tag.get('Key').split('/')[-1]
```

Comment:
Why its hard coded 'kubernetes.io/cluster/'  ?

Reply:
If we found the cluster with `sigs.k8s.io/cluster-api-provider-aws/cluster` need to convert `kubernetes.io/cluster` as it makes easy to identify.

Reply:
I dont think its the right way to handle it.
We need to do 2 searches one for sigs.k8s.io/cluster-api-provider-aws/cluster and one for kubernetes.io/cluster.
Maybe in the future there will be more options